source/portable/os/ota_os_freertos.c

```diff
@@ -145,7 +144,7 @@ OtaOsStatus_t OtaReceiveEvent_FreeRTOS( OtaEventContext_t * pEventCtx,
     {
         otaOsStatus = OtaOsEventQueueReceiveFailed;
 
-        LogError( ( "Failed to receive event from OTA Event Queue: "
+        LogError( ( "Failed to receive event or timeout from OTA Event Queue: "
```

Comment:
Now that queue receive has a timeout value can this cause noisy logs when the OTA agent is idle?

Reply:
Set it to LogDebug to reduce noisy logs.